testing/trino-testing/pom.xml

```diff
@@ -46,6 +46,13 @@
         <dependency>
             <groupId>io.trino</groupId>
             <artifactId>trino-testing-services</artifactId>
+            <exclusions>
+                <exclusion>
+                    <!-- jmh-core not allowed as a compile time dependency -->
```

Comment:
Not allowed in what sense?
who checks that?

Reply:
```
[INFO] --- dependency-scope-maven-plugin:0.10:check (default) @ trino-testing ---
[ERROR] Found a problem with test-scoped dependency org.openjdk.jmh:jmh-core
  Scope compile was expected by artifact: io.trino:trino-testing-services:357-SNAPSHOT
```

Reply:
It seems I should as an alternative be able to change scope for jmh dependencies in trino-testing to compile. 
But then I get: 
[WARNING] Unused declared dependencies found:
[WARNING]    org.openjdk.jmh:jmh-core:jar:1.20:compile
[WARNING]    org.openjdk.jmh:jmh-generator-annprocess:jar:1.20:compile

Reply:
I changed the comment to `<!-- conflicts with test-scoped dependecies declared below -->`